### PR TITLE
fix(runtime): check input path exists

### LINKED ISSUE #10: Error when input does not exist
The action currently Errors when the input does not exist, this causes the step to fail. Instead log that it does not exist and exit.

### PR DESCRIPTION
**Pull request by blake-newman**
check input path exists; if it does not warn and exit

closes GH-10

### CODE REVIEW

**Review by csi-lk — APPROVED** [review]

File: src/index.ts [hunk 1]
```diff
@@ -7,8 +7,12 @@ import fs from 'fs'
 async function main(): Promise<void> {
   const { token, title, path } = getInputs()
 
-  const github = new Github(token)
+  if (!fs.existsSync(path)) {
+    core.warning(`Input file ${path} does not exist`)
+    return
```
Comment (ycmjason) [inline src/index.ts:12]: `process.exit(1)`?

Comment (blake-newman) [reply src/index.ts:12]: If it was a node thing yup; but as it's an action we don't want to exit the process.

**Comment by ycmjason** [comment]
> 🎉 This PR is included in version 1.0.7 🎉
> 
> The release is available on [GitHub release](https://github.com/Attest/annotations-action/releases/tag/v1.0.7)
> 
> Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot 📦🚀

so cool

**Pull request merged**